draft-ietf-ice-pac.xml

```diff
@@ -177,22 +177,22 @@
         needs to wait enough time to allow peer-reflexive candidates to be
         discovered. Accordingly, when full ICE implementations begin their
         ICE processing, as described in <xref target="RFC8445" />, Section
-        6.1, the ICE agent MUST set a timer, and its duration SHOULD be
-        equal to the ICE agent's connectivity check timeout.
+        6.1, the agent MUST set a timer, and its duration SHOULD be
+        equal to the agent's Binding request transaction timeout.
```

Comment:
"connectivity check transaction timeout"

Reply:
I can do that.